tests/modules/programs/neomutt/neomutt-not-primary.nix

```diff
@@ -0,0 +1,25 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  imports = [ ../../accounts/email-test-accounts.nix ];
+
+  config = {
+    accounts.email.accounts = {
+      "hm@example.com".maildir = null;
+      hm-account.neomutt.enable = true;
+    };
+
+    programs.neomutt.enable = true;
+
+    nixpkgs.overlays =
+      [ (self: super: { neomutt = pkgs.writeScriptBin "dummy-neomutt" ""; }) ];
+
+    nmt.script = ''
+      assertFileExists home-files/.config/neomutt/neomuttrc
+      # it would be nice to test this - not sure how
+      # neomutt -e "push '<exit>'"
```

Comment:
Not really possible without pulling in the neomutt package as a dependency.

You should assert on the file content, though.

Reply:
OK, added file content assert.